pkg/controller/nodes/nodes_controller.go

```diff
@@ -265,6 +311,23 @@ func (r *ReconcileNodes) removeNode(c *Cache, node string, oaFunc func(name stri
 	return nil
 }
 
+func (r *ReconcileNodes) updateNode(cache *Cache, nodeName string) error {
```

Comment:
Perhaps you want to use `c` for the `cache` variable as well, as it's now used on the rest of the PR?